README.md

```diff
@@ -31,7 +31,7 @@ This is usable as is in v0.2.x but in v0.3.0 and above, you need to add a file t
 
 ## PekkoDevelocityPlugin
 
-This autoplugin enables automatic publishing of Build Scan® for every CI build, as well as for every local build executed by an authenticated Apache committer, to the [Apache Develocity](https://ge.apache.org) instance.
+This autoplugin enables automatic publishing of [Develocity](https://gradle.com/) Build Scan® for every CI build, as well as for every local build executed by an authenticated Apache committer, to the [Apache Develocity](https://ge.apache.org) instance.
```

Comment:
You will need to explain in detail how developers are going to authenticate and how they can opt out. No local builds can fail because a user has not set up auth for ge.apache.org.
I don't mind so much about CI builds but this mining of data from user laptops goes against the spirit of the ASF.

Reply:
I think this should sit behind an sbt setting that can default to true in CI (specifically use sbt's `insideCI.value`) and false otherwise if people use it locally?

Reply:
My understanding is that you need a develocity access key. I have raised an INFRA JIRA to have one set up for our GitHub CI builds. It looks like Develocity UI has a way to let logged in Develocity users to create their own access key for their own local builds. https://ge.apache.org/settings/access-keys

I guess that I have no major objection if people building on their own laptops have to set up the access key locally and that those who don't won't have their builds failing because they haven't done this.

Reply:
In that case its pretty simple, we can do

```
if (insideCI.value
  enableDevelocity
else if (sys.env.get("DEVELOCITY_ACCESS_KEY"))
  enableDevelocity
else
  dontEnable
```

Reply:
> My understanding is that you need a develocity access key.

@pjfanning Correct. An access key is needed if you want a build scan to be published (if interested, see [this section](https://docs.gradle.com/enterprise/sbt-plugin/#authenticating) in our user manual for details). The following condition `Publishing.onlyIf(_.authenticated)` is ensuring that a build scan is attempted to be published iff an access key is configured. No further logic is required.

Reply:
> No local builds can fail because a user has not set up auth for ge.apache.org

@pjfanning I believe this is no longer perceived as an issue, but I wanted to confirm that no local builds will fail if a user hasn't set up an access key.

> I guess that I have no major objection if people building on their own laptops have to set up the access key locally and that those who don't won't have their builds failing because they haven't done this.

That's correct. It's opt-in to publish a build scan to the Apache Develocity instance (https://ge.apache.org/). Also, for the avoidance of doubt, no information is transmitted to Gradle. If a build scan is publish to the Apache Develocity instance, that's where the information will be living (until the data retention period is over, at which point they are deleted for good).